src/DDserver.cpp

```diff
@@ -187,72 +186,72 @@ namespace DoubleD
 
         CROW_ROUTE(app, "/status")
             ([&](const crow::request& req) {
+            
             crow::json::wvalue x;
+            x["servername"] = DDserver::m_server_name;
 
             if (req.url_params.get("auth") == nullptr)
             {
-                x[DDserver::m_server_name] = "no key";
+                x["status"] = "no api key";
                 return crow::response(400, x);
             }
             else if (DDserver::m_api_key != req.url_params.get("auth"))
             {
-                x[DDserver::m_server_name] = "invalid key";
+                x["status"] = "invalid key";
                 return crow::response(401, x);
             }
 
+            x["status"] = "ok";
+
+            // Create an empty vector.
+            // Crow will convert this to a JSON list.
+            // The list will be overwritten if the server contains locks.
+            std::vector<std::string> jsonList;
+            x["locks"] = jsonList;
+
             DDserver::m_storageMutex.lock();
-            if (DDserver::m_lockVector.size() > 0)
-            {
-                for (long unsigned int i = 0; i < DDserver::m_lockVector.size(); i++) {
-                    x[DDserver::m_server_name][i]["name"] = DDserver::m_lockVector[i].m_getName();
-                    x[DDserver::m_server_name][i]["token"] = DDserver::m_lockVector[i].m_getSessionToken();
-                    x[DDserver::m_server_name][i]["remaining"] = DDserver::m_lockVector[i].m_timeLeft();
-                }
-                DDserver::m_storageMutex.unlock();
-                return crow::response(200, x);
-            }
-            else
-            {
-                x[DDserver::m_server_name][0] = "empty";
```

Comment:
Now we return an actual empty list instead of writing "empty", which is much better, I think